### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by kensens**
some small misc. fixes

### CODE REVIEW

File: crossbeam/xm_train.py [hunk 1]
```diff
@@ -69,15 +72,30 @@ def main(argv) -> None:
...
     async def make_job(work_unit, **kwargs):
       args = deepcopy(executable_args)
-      args['config.save_dir'] = f'{save_dir}/{work_unit.work_unit_id}'
+      sweep_str_parts = []
+      for k, v in kwargs.items():
+        if k.startswith('config.'):
+          k = k[len('config.'):]
+        sweep_str_parts.append(f'{k}={v!r}')
+      sweep_str = ','.join(sweep_str_parts)
+      args['config.save_dir'] = f'{save_dir}/{work_unit.work_unit_id}_{sweep_str}'
```
Comment (Hanjun-Dai) [inline crossbeam/xm_train.py:81]: one minor issue: the 'save_dir/work_unit.work_unit_id' is unique enough to identify a folder, as the save_dir itself is formatted with xm job id. Do you still prefer adding the sweep_str as part of the folder name?

**Review by Hanjun-Dai — APPROVED** [review]
thanks for fixing this! added a minor comment about formatting

File: crossbeam/xm_train.py (hunk 1 above)
Comment (kensens) [reply crossbeam/xm_train.py:81]: yes because the folder name is what's shown on TensorBoard

Comment (Hanjun-Dai) [reply crossbeam/xm_train.py:81]: got it, then this looks perfect!

**Pull request merged**